docs/release-notes/stable versions.md

```diff
@@ -0,0 +1,31 @@
+---
+id: stable-versions
+title: Mia-Platform stable versions
+sidebar_label: Stable versions
+---
+
+A **stable release** is a version that has been thoroughly tested and is therefore as reliable as possible.
+
```

Comment:
I'd remove this line, since also other releases are well tested before the release

Reply:
rephrased a little bit to remove the testing topic